.gitattributes

```diff
@@ -0,0 +1,11 @@
+# Generated files
```

Comment:
I'd prefer to be able to see changes in these, as a sanity check rather, than them being opaque black boxes.

# Before

```diff
diff --git a/include/branches.csv b/include/branches.csv
index bfec9da..fbf0153 100644
--- a/include/branches.csv
+++ b/include/branches.csv
@@ -1,5 +1,5 @@
 Branch,Schedule,Status,First release,End of life,Release manager
-main,:pep:`693`,feature,*2023-10-02*,*2028-10*,Thomas Wouters
+main,:pep:`693`,feature,*2023-10-02*,*2028-11*,Thomas Wouters
 3.11,:pep:`664`,bugfix,2022-10-24,*2027-10*,Pablo Galindo Salgado
 3.10,:pep:`619`,bugfix,2021-10-04,*2026-10*,Pablo Galindo Salgado
 3.9,:pep:`596`,security,2020-10-05,*2025-10*,Łukasz Langa
diff --git a/include/end-of-life.csv b/include/end-of-life.csv
index be9fe1b..f2aa404 100644
--- a/include/end-of-life.csv
+++ b/include/end-of-life.csv
@@ -1,5 +1,5 @@
 Branch,Schedule,Status,First release,End of life,Release manager
-3.6,:pep:`494`,end-of-life,2016-12-23,2021-12-23,Ned Deily
+3.6,:pep:`494`,end-of-life,2016-12-23,2021-12-31,Ned Deily
 3.5,:pep:`478`,end-of-life,2015-09-13,2020-09-30,Larry Hastings
 3.4,:pep:`429`,end-of-life,2014-03-16,2019-03-18,Larry Hastings
 3.3,:pep:`398`,end-of-life,2012-09-29,2017-09-29,"Georg Brandl, Ned Deily (3.3.7+)"
diff --git a/include/release-cycle.svg b/include/release-cycle.svg
index 53714a1..65eed8b 100644
--- a/include/release-cycle.svg
+++ b/include/release-cycle.svg
@@ -1011,7 +1011,7 @@
             y="405.0"
             font-size="18"
         >
-            Python 3.12
+            Python 3.20
         </text>
 
         <!-- Colourful blob with a label -->
```

# After

```diff
diff --git a/include/branches.csv b/include/branches.csv
index bfec9da..fbf0153 100644
Binary files a/include/branches.csv and b/include/branches.csv differ
diff --git a/include/end-of-life.csv b/include/end-of-life.csv
index be9fe1b..f2aa404 100644
Binary files a/include/end-of-life.csv and b/include/end-of-life.csv differ
diff --git a/include/release-cycle.svg b/include/release-cycle.svg
index 53714a1..65eed8b 100644
Binary files a/include/release-cycle.svg and b/include/release-cycle.svg differ
```